pkg/diff/diff.go

```diff
@@ -851,6 +851,20 @@ func (t *TableDiff) UpdateSummaryInfo(ctx context.Context) chan bool {
 	return stopUpdateCh
 }
 
+// GetTableInfo returns table information.
+func getTableInfo(ctx context.Context, db dbutil.QueryExecutor, schemaName string, tableName string) (*model.TableInfo, error) {
```

Comment:
it's removed in tidb report in here https://github.com/pingcap/tidb/pull/56348/files#diff-d4ddbee532a16bfb35282bac72a6e53bdd9964ba2b658d02ed28ce1a834d7b8bL34

so copied here